ignite/handlers/param_scheduler.py

```diff
@@ -226,6 +227,256 @@ def plot_values(cls, num_events: int, **scheduler_kwargs: Mapping) -> Any:
         return ax
 
 
+class StateParameterScheduler(ParamScheduler):
+    """An abstract class for updating a state parameter during
+    training not belonging to an Optimizer parameter group.
```

Comment:
```suggestion
    """An abstract class for updating a state parameter during
    the run.
```
There is no params in the `engine.state` that belong to optimizer param group